keyboards/ep/96/readme.md

```diff
@@ -0,0 +1,15 @@
+# ep96
+
+![ep96](https://imgur.com/PZ2C7a8)
+
+A simple 96 key keyboard
+
+Keyboard Maintainer: Keyboard Maintainer: [Elliot Powell](https://github.com/e11i0t23), [/u/e11i0t23](https://reddit.com/u/e11i0t23)  
+Hardware Supported: EP96 PCB
```

Comment:
Formatting is a bit off, take a look at https://github.com/qmk/qmk_firmware/blob/9586d720f8a107d6005d88a2b8727bc040f5eb31/keyboards/ep/96/readme.md

```suggestion
Hardware Supported: EP96 PCB   
```